components/collapsible-panel/collapsible-panel-group.js

```diff
@@ -58,6 +58,8 @@ class CollapsiblePanelGroup extends SkeletonMixin(LitElement) {
 	_updatePanelAttributes() {
 		if (!this._panels || this._panels.length === 0) return;
 
+		if (this._panels?.[0]?.type !== 'inline') return;
```

Comment:
If we ever want to update panel attributes for the other cases, this will need to be reworked a bit. But I think the performance bump of short circuiting iteration is worth it.

Reply:
That's true, the optional chain is redundant.

I think I'd rather remove it and leave them both as explicit checks, since they're logically doing different things, and it's _somewhat_ likely that we could remove/rework the `inline` check.